logger.py

```diff
@@ -0,0 +1,13 @@
+import logging
+
+
+def get_logger():
+    logger = logging.getLogger("jiminy-modeller")
+    if not logger.handlers:
+        ch=logging.StreamHandler()
+        formatter = logging.Formatter('%(asctime)s - %(name)s - %(levelname)s-%(message)s')
+        ch.setFormatter(formatter)
+        logger.addHandler(ch)
+        logger.setLevel(logging.DEBUG)
+
+        return logger
```

Comment:
this shouldn't be indented, it will cause a `None` return if the `if` statement fails

Reply:
Thanks for that.